tinyagentos/routes/agent_registry.py

```
@@ -691,3 +704,60 @@ async def reactivate_agent(
+    if not user.is_admin:
+        raise HTTPException(status_code=403, detail="forbidden")
+    store = _get_store(request)
+    return {"tree": await store.get_org_tree()}
+
+
+@router.put("/api/agents/{canonical_id}/org")
+async def update_org_fields(
+    request: Request,
+    canonical_id: str,
+    body: OrgUpdateRequest,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)